version.gradle

```diff
@@ -2,7 +2,7 @@ ext {
     // ArcGIS Maps SDK for Kotlin version
     arcgisVersion = '200.3.0-4075'
     // ArcGIS Maps SDK for Kotlin Toolkit version
-    arcgisToolkitVersion = '200.3.0-4075'
+    arcgisToolkitVersion = '200.3.0-1'
```

Comment:
shouldn't this be `200.3.0` ?

Reply:
`200.3.0-1` is the published toolkit version
https://devtopia.esri.com/runtime/kotlin/issues/3358#issuecomment-4463792